app/exporters/odt/cv.rb

```diff
@@ -39,16 +41,21 @@ def insert_general_sections(report)
       report.add_field(:version, '1.0')
       report.add_field(:comment, 'Aktuelle Ausgabe')
     end
-    # rubocop:enable Metrics/AbcSize
 
+    # rubocop:enable Metrics/AbcSize
     # rubocop:disable Metrics/AbcSize
-    def insert_personalien(report)
+    # rubocop:disable Metrics/LineLength
+    def insert_personalien(report, anon)
       report.add_field(:title, person.title)
-      report.add_field(:birthdate, Date.parse(person.birthdate.to_s).strftime('%d.%m.%Y'))
-      report.add_field(:nationalities, nationalities)
-      report.add_field(:email, person.email)
-      report.add_image(:profile_picture, person.picture.path) if person.picture.file.present?
+      unless anon
+        report.add_field(:birthdate, Date.parse(person.birthdate.to_s)
+                                         .strftime('%d.%m.%Y'))
+      end
+      report.add_field(:nationalities, nationalities) unless anon
+      report.add_field(:email, person.email) unless anon
+      report.add_image(:profile_picture, person.picture.path) unless anon || person.picture.file.blank?
```

Comment:
why use unless anon as a block first and then use it in guard close after as well? Just put everything into the unless anon block